README.md

```diff
@@ -15,18 +15,18 @@ If you have any questions about using the template, please contact the transform
 # How to Produce a Publication *Summary* in Word using RMarkdown
 
 * To download the necessary files, click on *Clone or download -> Download ZIP* and save it to the folder of your choice. 
```

Comment:
Formatting is off here, might need to have a blank line between the end of the sentence and the image?